### PR TITLE
496 custom column chart tooltip

### LINKED ISSUE #496: Add ng/ts support for ep issue #1909
**Describe the bug**
Need to add data to custom tooltip callback with column chart.

**To Reproduce**
see [EP Issue #1909](https://github.com/infor-design/enterprise/issues/1909)

### PR DESCRIPTION
**Pull request by pwpatton**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Added input for custom tooltip for column charts

**Related github/jira issue (required)**:
closes #496

### CODE REVIEW

File: projects/ids-enterprise-ng/src/lib/column/soho-column.d.ts [hunk 1]
```diff
@@ -6,11 +6,20 @@
  */
 
 type SohoColumnType = 'column' | 'column-grouped' | 'column-stacked' | 'column-positive-negative' | 'column-positive-negative';
+
+/**
+ * The funcito ntype used for a custom tooltip in SohoChartOptions and SohoChartData.
```
Comment (fitzorama) [inline projects/ids-enterprise-ng/src/lib/column/soho-column.d.ts]: function type

File: projects/ids-enterprise-ng/src/lib/column/soho-column.component.ts [hunk 2]
```diff
@@ -147,6 +147,16 @@ export class SohoColumnComponent implements AfterViewInit, AfterViewChecked, OnD
     }
   }
 
+  /** Settings for the chart tooltip. */
+  @Input() set tooltip(tooltip: string | SohoChartCustomTooltipFunction) {
```
Comment (fitzorama) [inline projects/ids-enterprise-ng/src/lib/column/soho-column.component.ts]: chart?

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 ### 5.4.0 Features
 
 - `[Personalize]` - Changed soho-personalize directive to include theme and personalization color information. `PWP` ([#493](https://github.com/infor-design/enterprise-ng/pull/493))
+- `[Column Chart]` - Added custom or standard tooltip to inputs. `PWP` ([#496](https://github.com/infor-design/enterprise-ng/issues/496))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md]:
```suggestion
- `[Column Chart]` - Added the ability to use both dynamic or simple string based tooltips. `PWP` ([#496](https://github.com/infor-design/enterprise-ng/issues/496))
```

File: projects/ids-enterprise-ng/src/lib/column/soho-column.d.ts (hunk 1 above)
Comment (tmcconechy) [inline projects/ids-enterprise-ng/src/lib/column/soho-column.d.ts]:
```suggestion
 * The function type used for a dynamic tooltip in SohoChartOptions and SohoChartData.
```

**Pull request merged**